script/generate-release-notes.js

```diff
@@ -52,10 +58,11 @@ async function run() {
     token,
   })
 
-  const user = await octokit.users.get()
+  const user = await octokit.users.get({})
   const me = user.data.login
 
   console.log(`✅ Token found for ${me}`)
+  // @ts-ignore
```

Comment:
This one was the only really complex case I couldn't easily workaround:

<img width="522" src="https://user-images.githubusercontent.com/359239/50053543-4c6e2400-010d-11e9-8df0-165a7148f2e4.png">

I might submit a PR upstream to `@octokit/rest` to add this if I can confirm this is sent on every API response.